spec/unit/prometheus_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'vmpooler/statsd'
+
+describe 'prometheus' do
+
+  let(:providers) do
+    Vmpooler::Providers.new
+  end
+
+
+
+  describe '#initialise' do
+
+    it 'XXXX' do
+    end
+  end
+
+  describe '#setup_prometheus_metrics' do
+
+    it 'XXXX' do
+    end
+  end
+
+  describe '#find_metric' do
+
+    it 'XXXX' do
+    end
+  end
```

Comment:
I assume you are going to clean this up....

Reply:
Yes - the commit header has a clear TBD on this.